flink-runtime-web/src/test/resources/rest_api_v1.snapshot

```diff
@@ -1628,7 +1628,7 @@
           "type" : "array",
           "items" : {
             "type" : "object",
-            "id" : "urn:jsonschema:org:apache:flink:runtime:rest:messages:JobVertexDetailsInfo:VertexTaskDetail",
+            "id" : "urn:jsonschema:org:apache:flink:runtime:rest:messages:job:SubtaskExecutionAttemptDetailsInfo",
```

Comment:
Is this a backwards compatible change @zentol? If not, then I guess my reccommendation to replace `VertexTaskDetail` with `SubtaskExecutionAttemptDetailsInfo` was wrong and would need to be reverted.

Reply:
yes, this is fine.´; it's just schema meta-data, which isn't relevant for backwards-compatibility.